pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>3.56</version>
+        <version>4.46</version>
```

Comment:
So the build is failing because it can't find the bom for 
<jenkins.verison>1.609.1</jenkins.version>

jenkins 1.x is really really old, I think to use the newer version that is supported

```
[2022-08-21T15:59:10.910Z] [ERROR] Non-resolvable import POM: Could not find artifact org.jenkins-ci.main:jenkins-bom:pom:1.609.1 in repo.jenkins-ci.org (https://repo.jenkins-ci.org/public/) @ org.jenkins-ci.plugins:plugin:4.46, C:\Jenkins\workspace\ins_docker-java-api-plugin_PR-39@tmp\m2repo\org\jenkins-ci\plugins\plugin\4.46\plugin-4.46.pom, line 122, column 19
```

Looking at https://repo.jenkins-ci.org/public/org/jenkins-ci/main/jenkins-bom/ it looks like there are boms for 2.164.1 and above.